.github/workflows/full.yml

```diff
@@ -608,6 +644,39 @@ jobs:
           registry: docker.io
           username: dyrectorio
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Golang caches
+        uses: actions/cache@v3
+        with:
+          path: |
+            $GOPATH/cache
+            $GOPATH/pkg/mod
+          key: ${{ runner.os }}-golang-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-golang-
+      # fixes: fatal: unsafe repository
+      - name: Adding workspace
+        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+      - name: Load go mod
```

Comment:
Mixin `go` ang `Golang`, be consistent please.

Reply:
Golang is the language, go.mod is the lock file, but could be more verbose.

Reply:
I think it's fine